pkg/volume/csi/csi_client.go

```diff
@@ -117,7 +118,9 @@ func (c *csiDriverClient) AssertSupportedVersion(ctx grpctx.Context, ver *csipb.
 	vers := rsp.GetSupportedVersions()
 	glog.V(4).Info(log("driver reports %d versions supported: %s", len(vers), versToStr(vers)))
 
-	for _, v := range vers {
+	driverVersions := make([]string, len(vers))
+	for i, v := range vers {
+		driverVersions[i] = verToStr(v)
```

Comment:
there exists a "versToStr" function in this file that does this

Reply:
@lpabon function `versToStr` returns a string value of all supported versions https://github.com/lpabon/kubernetes/blob/7223f71257d5569ee60cdb2ac3e6e11dbc82e197/pkg/volume/csi/csi_client.go#L240 already.  Try to use it or update it if there is a problem.

Reply:
Nice!, I'll make the change